lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -144,21 +164,15 @@ try {
         $result.changed = $true
     }
     elseif( ($state -eq "present") -and ($exists -eq $true) ) {
-        if ($task.Description -eq $description -and $task.TaskName -eq $name -and $task.TaskPath -eq $path -and $task.Actions.Execute -eq $execute -and $taskState -eq $enabled -and $task.Principal.UserId -eq $user) {
```

Comment:
As far as I could tell the `true` codepath never gets executed even though task details don't change.  In addition, I realized this never checks to see if the trigger ever changes.